bumpversion/cli.py

```diff
@@ -352,9 +353,11 @@ def _load_configuration(config_file, explicit_config, defaults):
 
             if "replace" not in section_config:
                 section_config["replace"] = defaults.get("replace", "{new_version}")
-
-            files.append(ConfiguredFile(filename, VersionConfig(**section_config)))
-
+            if section_type.get("file") == "glob":
```

Comment:
Please keep an empty line in between
```suggestion

            if section_type.get("file") == "glob":
```